contracts/messaging/gateway/FuelERC20Gateway.sol

```diff
@@ -125,12 +169,35 @@ contract FuelERC20Gateway is
         //reduce deposit balance and transfer tokens (math will underflow if amount is larger than allowed)
         _deposits[tokenId][fuelTokenId] = _deposits[tokenId][fuelTokenId] - amount;
         IERC20Upgradeable(tokenId).safeTransfer(to, amount);
+
+        //emit event for successful token withdraw
+        emit Withdrawal(bytes32(uint256(uint160(to))), tokenId, fuelTokenId, amount);
     }
 
     ////////////////////////
     // Internal Functions //
     ////////////////////////
 
+    /// @notice Deposits the given tokens to an account or contract on Fuel
```

Comment:
```suggestion
    /// @notice Deposits the given tokens to an address or contract on Fuel
```

Reply:
Address is a more generic term (a contract has an address to). Account seems to be the typical term used. I hear the phrase EOA a lot, which I believe means "externally owned account".

Reply:
Sorry, was getting mixed up and thinking about Fuel, where `Address` and `ContractId` are totally unique namespaces.
Agree that `account` is fine in this context.